src/main/features/dashboard/claims-state-machine/initial-transitions.ts

```diff
@@ -45,11 +45,11 @@ export function initialTransitions (claim: Claim): StateMachine {
         from: [InitialStates.INIT, InitialStates.NO_RESPONSE],
         to: InitialStates.HWF_APPLICATION_PENDING
       },
-      {
-        name: 'checkHwfIntrestReCalculate',
-        from: [InitialStates.INIT, InitialStates.NO_RESPONSE, InitialStates.HWF_AWAITING_RESPONSE_HWF],
-        to: InitialStates.HWF_Intrest_Recalculate
-      },
+      // {
+      //   name: 'checkHwfIntrestReCalculate',
```

Comment:
If this code is no longer required (removed as part of your story), please remove it instead of commenting it.